src/lib/components/ui/Modal/Modal.jsx

```diff
@@ -37,9 +62,19 @@ class Modal extends React.Component {
   }
 
   pressEscapeHandler(e) {
+    const { actions } = this.props;
+
     if (e.keyCode === 27 && this.props.closeHandler) {
       this.props.closeHandler();
     }
+
+    if (e.keyCode === 13 && e.target.nodeName !== 'BUTTON') {
```

Comment:
Does it also handle the numpad enter? I don't have such keyboard so I cannot test it.

Reply:
It does, there is no difference.